react/features/recording/components/Recording/StartRecordingDialogContent.js

```diff
@@ -52,6 +52,11 @@ type Props = {
      */
     t: Function,
 
+    /**
+     *  The color theme to be used (native only).
+     */
+    theme: ?string,
```

Comment:
why the inconsistency? shouldn't all dialogs be rendered equal? if theming support is required, can we extract it to its own commit then?

Reply:
Actually, that was intended for backward compatibility for web, but then I just found out that it's not really needed, so removed it. Thanks!